stdlib/public/runtime/HeapObject.cpp

```diff
@@ -98,7 +92,18 @@ static HeapObject *_swift_allocObject_(HeapMetadata const *metadata,
   return object;
 }
 
-auto swift::_swift_allocObject = _swift_allocObject_;
+HeapObject *swift::swift_allocObject(HeapMetadata const *metadata,
+                                     size_t requiredSize,
+                                     size_t requiredAlignmentMask) {
+  if (SWIFT_UNLIKELY(_swift_allocObject))
+    return _swift_allocObject(metadata, requiredSize, requiredAlignmentMask);
+  else
```

Comment:
The `else` is unnecessary, and similar throughout.

Reply:
Thanks, updated.